src/transform/use_indexes.rs

```diff
@@ -150,31 +142,19 @@ impl FilterEqualLiteral {
 pub struct FilterLifting;
 
 impl crate::Transform for FilterLifting {
-    fn transform(
-        &self,
-        relation: &mut RelationExpr,
-        indexes: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
-    ) {
-        self.transform(relation, indexes);
+    fn transform(&self, relation: &mut RelationExpr, state: &mut TransformArgs) {
+        self.transform(relation, state);
     }
 }
 
 impl FilterLifting {
-    pub fn transform(
-        &self,
-        relation: &mut RelationExpr,
-        indexes: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
-    ) {
+    pub fn transform(&self, relation: &mut RelationExpr, state: &mut TransformArgs) {
         relation.visit_mut(&mut |e| {
-            self.action(e, indexes);
+            self.action(e, state);
```

Comment:
Same as above, but up for discussion if you disagree.

Reply:
Made these changes, but this file doesn't get built and so has a bunch of compilation errors unrelated to this change.